stream-chat-android-compose/src/main/res/values-en/strings.xml

```diff
@@ -37,6 +37,12 @@
     <string name="stream_compose_user_status_last_seen">Last seen %1$s</string>
     <string name="stream_compose_user_status_last_seen_just_now">Last seen just now</string>
 
+    <!-- Message List Header -->
+    <plurals name="stream_compose_message_list_header_typing_users">
+        <item quantity="one">%s is typing</item>
+        <item quantity="other">%s and %d more are typing</item>
```

Comment:
What happens if we have many people typing? E.g. thousands?

Will the text still fit the subtitle width? 

Some apps (like discord) move to "Several people are typing" or something in case there are more than X number of people.

Maybe that can also be an option for us?

Reply:
I hope the channel type has typing indicators disabled when thousands of people regularly type at the same time, huge waste of resources to propagate those events 😃

Jokes aside, that's a sensible idea. I'd go with different phrasing, I feel like Slack owns those specific words (it's the [title of their blog](https://slack.com/blog#:~:text=Several%20People%20Are%20Typing), for example.

Reply:
Hm, I've added this to my TODO list.